encode/h264encode.c

```diff
@@ -174,6 +174,11 @@ static unsigned int SyncPictureTicks=0;
 static unsigned int SavePictureTicks=0;
 static unsigned int TotalTicks=0;
 
+//Default entrypoint for Encode
+static VAEntrypoint requested_entrypoint = -1;
+static VAEntrypoint selected_entrypoint = -1;
+static int lp_option = 0;
```

Comment:
lp_option is used in process_cmdline() only, so you needn't declare it here.

Reply:
done